src/components/fields/file-upload/file-upload-manager.ts

```diff
@@ -234,6 +234,10 @@ const FileUploadManager = (props: IProps) => {
 			rawFile?.size || fileToInject.uploadResponse?.["fileSize"]
 		);
 
+		let size = rawFile?.size || fileToInject.uploadResponse?.["fileSize"] || 0;
+		if (isNaN(size)) {
+			size = 0;
+		}
```

Comment:
since there's also some validation on the file size being done, the sanitisation can be done first and used as the param for `validateFileSize` and file data

something like `"1024" > 1` will evaluate to true and return the size error